### DISCUSSION TITLE
Custom Text between image preview and carousel

### DISCUSSION THREAD

**Discussion by yosiasz** [MurhafSousli/ngx-gallery#578] [Ideas]
Greetings and thanks for this amazing plugin

I was wondering if there is a way to add custom text between image preview and carousel instead of above the image

Current
<img width="540" alt="image" src="https://github.com/MurhafSousli/ngx-gallery/assets/900951/5b3570ce-b238-43e3-af78-7583d46a6f5c">

Desired
<img width="666" alt="image" src="https://github.com/MurhafSousli/ngx-gallery/assets/900951/00e5693b-f254-4b4f-88dd-a3477c0eb789">

Thanks!

**Comment by MurhafSousli** [comment]
The layer already has absolute position give a class name and apply

```css
.item-text {
    position: absolute;
    bottom: 0;
    left: 0;
    right: 0;
}
```

```html
<gallery>
  <ng-container *galleryImageDef="let item; let active = active">
    <div *ngIf="active" class="item-text">
      {{ item?.alt }}
    </div>
  </ng-container>
</gallery>
```
